### DISCUSSION TITLE
DismissPopupBar with swipe animation

### DISCUSSION THREAD

**Discussion by Alisher9** [LeoNatan/LNPopupController#491] [General] [answered]
Hi all,
is it possible to dismissPopupBar with a down swipe like on Youtube?
Thanks in advance)

**Comment by LeoNatan** [comment, marked as answer]
Hello,
This isn't supported. You can implement this on your own by adding a gesture recognizer, but you need to be careful with gesture recognizer conflicts between yours and the framework ones.
